src/ventus/wm/window.js

```diff
@@ -486,8 +489,23 @@ function(Emitter, Promise, View, WindowTemplate) {
 		},
 
 		move: function(x, y) {
-			this.x = x;
-			this.y = y;
+			var max_x = window.innerWidth - this._wm.margin.right;
+			var max_y = window.innerHeight - this._wm.margin.bottom;
```

Comment:
Is it ok to assume Ventus' size is window's size? What if Ventus container is a panel using just half screen?